apps/web/src/views/Farms/components/FarmTable/FarmTable.tsx

```diff
@@ -149,8 +147,7 @@ const FarmTable: React.FC<React.PropsWithChildren<ITableProps>> = ({ farms, cake
   const tableWrapperEl = useRef<HTMLDivElement>(null)
   const { query } = useRouter()
 
-  const { data: farmV2 } = useFarmsV2Data()
-  const { totalRegularAllocPoint, cakePerBlock } = farmV2 || { totalRegularAllocPoint: null, cakePerBlock: null }
+  const { totalRegularAllocPoint, regularCakePerBlock: cakePerBlock } = useFarms()
   const totalMultipliersV2 = totalRegularAllocPoint
     ? (ethersToBigNumber(totalRegularAllocPoint).toNumber() / 10).toString()
```

Comment:
again this follows the same issue i was describing above with the v2 useFarms not returning undefined for cakePerBlock and totalRegularAllocPoint datas.